tests/core/Jalasoft.TeamUp.Resumes.Core.Tests/GetResumesByNameCoreTests.cs

```diff
@@ -0,0 +1,162 @@
+                    {
+                        FirstName = "Guido",
+                        LastName = "Castro",
+                        Birthdate = DateTime.Now,
+                        Picture = "picture.jpg"
+                    },
+                    Summary = "I'm a great developer :-D",
+                    Skills = new List<Skill>()
+                    {
+                        new Skill
+                        {
+                            Id = 1,
+                            Name = "C#"
+                        },
+                        new Skill
+                        {
+                            Id = 2,
+                            Name = "Javascript"
+                        },
+                    }
+                },
+                new Resume()
+                {
+                    Id = 2,
+                    Title = "Marcelo Ruiz",
+                    Contact = new Contact
+                    {
+                        Address = "16 street",
+                        Email = "marcelo.ruiz@fundacion-jala.org",
+                        Phone = 123456789,
+                    },
+                    CreationDate = DateTime.Now,
+                    LastUpdate = DateTime.Now,
+                    Person = new Person
+                    {
+                        FirstName = "Marcelo",
+                        LastName = "Ruiz",
+                        Birthdate = DateTime.Now,
+                        Picture = "picture.jpg"
+                    },
+                    Summary = "I'm a great ux :-D",
+                    Skills = new List<Skill>()
+                    {
+                        new Skill
+                        {
+                            Id = 1,
+                            Name = "Figma"
+                        },
+                        new Skill
+                        {
+                            Id = 2,
+                            Name = "HTML"
+                        },
+                    }
+                }
+            };
+            return resumes;
+        }
+
+        [Fact]
+        public void GetResumesByName_NoItems_EmptyResult()
+        {
+            // Arrange
+            var stubEmptyResumeList = new List<Resume>();
+            this.mockRepository.Setup(repository => repository.GetByName("Gustavo")).Returns(stubEmptyResumeList);
+
+            // Act
+            var result = this.resumesService.GetByName("Gustavo");
+
+            // Assert
+            Assert.Empty(result);
+        }
+
+        [Fact]
+        public void GetResumesByName_ItemsExist_ResumesArray()
+        {
+            // Arrange
+            this.mockRepository.Setup(repository => repository.GetAll()).Returns(GetTestResumes().ToList());
+
+            // Act
+            var result = this.resumesService.GetResumes();
+
+            // Assert
+            Assert.Equal(2, result.Length);
+        }
+
+        [Fact]
+        public void GetResume_ValidId_SingleResume()
```

Comment:
we are not receiving an ID the name should not be "validID"

Reply:
we probably don't need to test a single Resume in this case either since we return a Resume array